pkg/rbd/rbd_util.go

```diff
@@ -163,6 +163,30 @@ func rbdStatus(pOpts *rbdVolume, cr *util.Credentials) (bool, string, error) {
 	return false, output, nil
 }
 
+// rbdManagerTaskDelete adds a ceph manager task to delete an rbd image, thus deleting
+// it asynchronously. If command is not found returns a bool set to false
+func rbdManagerTaskDeleteImage(pOpts *rbdVolume, cr *util.Credentials) (bool, error) {
+	var output []byte
+
+	args := []string{"rbd", "task", "add", "remove",
+		pOpts.Pool + "/" + pOpts.RbdImageName,
+		"--id", cr.ID,
+		"--keyfile=" + cr.KeyFile,
+		"-m", pOpts.Monitors,
+	}
+
+	output, err := execCommand("ceph", args)
+	if err != nil {
+		if strings.Contains(string(output), "no valid command found") &&
```

Comment:
@ShyamsundarR if possible define these ( `no valid command found` ) as Error var/const.

Reply:
As a practice in other places in the code I/we have not done so, do we still want to change it? (searching for `strings.Contains` in the code shows up 7 instances)

Reply:
@ShyamsundarR the problem is that, if there changes or some manual errors in the code ( say a space or similar) can cause these code path to fail. Instead if we have well defined constant/vars it is less error prone. So , I would recommend to have it addressed.

Reply:
Fixed to use constant strings.

Reply:
Thanks!